notebooks/apify_haystack_rag_web_browser.ipynb

```diff
@@ -0,0 +1,479 @@
+{
```

Comment:
<p>Can you give more info around the how this actor works? something like "First, we need to understand how <a href="https://apify.com/apify/rag-web-browser" target="_blank">RAG Web Browser Actor</a> works and its output format. It accepts a search phrase or URL, performs a Google Search, crawls web pages from the top search results, cleans the HTML, and converts the content into text or Markdown"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>